lib/Text.js

```diff
@@ -68,7 +67,7 @@ export default class Text extends React.Component<TextProps> {
       <NativeText
         fill={extractBrush(props.fill, props)}
         opacity={extractOpacity(props)}
-        stroke={extractColor(props.stroke)}
+        stroke={props.stroke}
```

Comment:
This breaks the current API. Is it possible to keep `extractColor`?

Reply:
Yes, in that case extractColor will return the argument which was passed for iOS and for Android it can return Array of number. these also breaks the API, but the changes done for Android wont be needed

Reply:
I don't really want to break API for now. People will come from core `react-native`, and I want them to migrate as easily as possible.

Reply:
@Esemesek Code has been updated. Added Platform check only. keeping the API intact